src/components/Firestore/index.test.tsx

```diff
@@ -77,29 +81,32 @@ describe('FirestoreRoute', () => {
 
 describe('Firestore', () => {
   it('shows the top-level collections', async () => {
-    DatabaseApi.prototype.getCollections.mockResolvedValue([
-      fakeCollectionReference({ id: 'cool-coll' }),
-    ]);
-    const { getByText, queryByText } = render(
-      <MemoryRouter>
-        <Firestore config={sampleConfig} projectId={'foo'} />
-      </MemoryRouter>
+    let getCollections = makeDeferred<FakeCollectionReference[]>();
+    DatabaseApi.prototype.getCollections.mockReturnValue(
+      getCollections.promise
     );
+    let renderResult!: RenderResult;
+    act(() => {
```

Comment:
afaik render already is wrapped in an act:

https://github.com/testing-library/react-testing-library/blob/master/src/pure.js#L53-L59

Reply:
Done.